$NetBSD: patch-aa,v 1.6 2013/01/15 16:21:34 morr Exp $

- take build configuration from inherited pkgsrc variables
- install under $(DESTDIR)

--- Makefile.orig	2000-01-23 19:01:55.000000000 +0000
+++ Makefile
@@ -55,9 +55,9 @@ DROPALL=
 ####################
 # C compiler options
 # GNU gcc is recommended.
-CC=gcc
+#CC=gcc
 # Use these with gcc
-CFLAGS=-g -Wall -W -fno-common -pedantic
+#CFLAGS+= -Wall -W -fno-common -pedantic
 
 # If you don't have gcc, don't use -Wall -W -fno-common
 #CFLAGS=-g
@@ -77,23 +77,23 @@ CFLAGS=-g -Wall -W -fno-common -pedantic
 #CFLAGS+=-DTEST_GAME
 
 ## Initial linker options
-LDFLAGS=
+#LDFLAGS=
 
 ## Installation prefix
 # Adjust to taste. Stuff gets installed here.
-PREFIX=/usr/local
+#PREFIX=/usr/local
 #PREFIX=/usr/local/X11
 
 ## X11 location and options
 # for X11R6
 # Point this at your X tree.
-XDIR=/usr/X11R6
+XDIR=${X11BASE}
 #XDIR=/usr/local/X11
 #XDIR=/usr/openwin
 
 # try Xaw if you don't have Xaw3d
-#XAW=Xaw
-XAW=Xaw3d
+XAW=Xaw
+#XAW=Xaw3d
 
 XLIBS=-L$(XDIR)/lib -l$(XAW) -lXext -lXmu -lXt -lSM -lICE -lX11
 
@@ -106,7 +106,6 @@ CFLAGS+=$(XINC)
 # System V (Solaris, Irix, etc.) will probably want -lsocket -lnsl.
 #LIBS=-lsocket -lnsl
 # on other systems leave LIBS blank for now
-LIBS=
 
 # On some systems -DNARROWPROTO is needed for working Xaw scrollbars
 # This includes FreeBSD 3.x and recent Linux
@@ -208,16 +207,16 @@ libfriskAI.a: network.o aiDice.o game.o 
 	$(RANLIB) $@
 
 install: all
-	-(umask 022; mkdir -p -m755 $(LIBDIR) $(BINDIR) $(APPLOADDIR))
-	$(INSTALL_DATA) $(MAP) $(LIBDIR)
-	$(INSTALL_DATA) $(COUNTRY) $(LIBDIR)
-	$(INSTALL_DATA) $(HELP) $(LIBDIR)
-	$(INSTALL_BIN) risk $(BINDIR)
-	$(INSTALL_BIN) xfrisk $(BINDIR)
-	$(INSTALL_BIN) friskserver $(BINDIR)
-	$(INSTALL_BIN) aiDummy $(BINDIR)
-	$(INSTALL_BIN) aiConway $(BINDIR)
-	$(INSTALL_BIN) aiColson $(BINDIR)
+	-(umask 022; mkdir -p -m755 $(DESTDIR)$(LIBDIR) $(DESTDIR)$(BINDIR))
+	$(INSTALL_DATA) $(MAP) $(DESTDIR)$(LIBDIR)
+	$(INSTALL_DATA) $(COUNTRY) $(DESTDIR)$(LIBDIR)
+	$(INSTALL_DATA) $(HELP) $(DESTDIR)$(LIBDIR)
+	$(INSTALL_BIN) risk $(DESTDIR)$(BINDIR)
+	$(INSTALL_BIN) xfrisk $(DESTDIR)$(BINDIR)
+	$(INSTALL_BIN) friskserver $(DESTDIR)$(BINDIR)
+	$(INSTALL_BIN) aiDummy $(DESTDIR)$(BINDIR)
+	$(INSTALL_BIN) aiConway $(DESTDIR)$(BINDIR)
+	$(INSTALL_BIN) aiColson $(DESTDIR)$(BINDIR)
 
 
 # Other targets
@@ -225,13 +224,13 @@ Countries.risk World.risk: World.ppm bui
 	./buildmap World.ppm World.risk Countries.risk
 
 uninstall:
-	rm -rf $(LIBDIR)
-	rm -f  $(BINDIR)/risk
-	rm -f  $(BINDIR)/xfrisk
-	rm -f  $(BINDIR)/friskserver
-	rm -f  $(BINDIR)/aiDummy
-	rm -f  $(BINDIR)/aiConway
-	rm -f  $(BINDIR)/aiColson
+	rm -rf $(DESTDIR)$(LIBDIR)
+	rm -f  $(DESTDIR)$(BINDIR)/risk
+	rm -f  $(DESTDIR)$(BINDIR)/xfrisk
+	rm -f  $(DESTDIR)$(BINDIR)/friskserver
+	rm -f  $(DESTDIR)$(BINDIR)/aiDummy
+	rm -f  $(DESTDIR)$(BINDIR)/aiConway
+	rm -f  $(DESTDIR)$(BINDIR)/aiColson
 
 types.h: findtypes
 	./findtypes
